test/liquidation/liquidation.test.js

```diff
@@ -95,4 +88,31 @@ describe("Integration -- liquidation happy path", () => {
       )
     })
   })
+
+  describe("when deposit has changed the state outside the coverage pool during pending coverage pool auction", () => {
```

Comment:
Maybe we can shorten it to just `when deposit has been liquidated by someone else`?

Reply:
Fixed in 2c29566b2fba6d7c26e266b118d46d24f18e4d5c.